Library/Formula/snappystream.rb

```diff
@@ -14,9 +14,10 @@ class Snappystream < Formula
 
   depends_on "cmake" => :build
   depends_on "snappy"
+  depends_on "boost"
```

Comment:
Boost support in snappystream means that snappystream provides a few interfaces to work with boost iterators library. So this dependency is runtime. But it can be turned on/off by the WITH_BOOST_IOSTREAMS flag. Should we make it _:optional_?